.gitattributes

```diff
@@ -4,6 +4,7 @@ Makefile    text eol=lf
 .travis.yml text eol=lf
 .sh         text eol=lf
 .pl         text eol=lf
+.go         text eol=lf
```

Comment:
Please revert this, and set `core.autocrlf` to `true` in git, either local to your clone or globally.